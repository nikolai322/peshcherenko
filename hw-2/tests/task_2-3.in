abcd
2bc0
