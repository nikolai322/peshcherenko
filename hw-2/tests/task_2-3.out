No match found in the second string
1001
